java/src/org/simpledao/Utils.java

```diff
@@ -24,10 +24,11 @@ public class Utils
      * @return a map of properties
      * @deprecated Use ReflectionUtils.getBeanPropertyMap instead
      */
-    public static Map<String,String> getBeanPropertyMap(Object bean)
+    /*public static Map<String,String> getBeanPropertyMap(Object bean)

```

Comment:
Since anyone that uses this version of the library will notice right away that this method isn't available, maybe we can consider removing these comments?